docs/non-normative/groups/system/design-philosophy.md

```diff
@@ -0,0 +1,94 @@
+
+## Two Class Design Strategy
+
+Relevant discussions: [\#1403 (particular comment)](https://github.com/open-telemetry/semantic-conventions/issues/1403#issuecomment-2368815634)
+
+We are considering two personas for system semconv instrumentation. If we have a piece of instrumentation, we decide which persona it is meant for and use that to make the decision for how we should name/treat that piece of instrumentation.
+
+### General Class: A generalized cross-platform use case we want any user of instrumentation to be able to easily access
+
+When instrumentation is meant for the General Class, we will strive to make the names and examples as prescriptive as possible. This instrumentation is what will drive the most important use cases we really want to cover with the system semantic conventions. Things like dashboards, alerts, and broader o11y setup tutorials will largely feature General Class instrumentation covering the [basic use cases](./use-cases.md) we have laid out as a group. We want this instrumentation to be very clear exactly how and when they should be used. General Class instrumentation will be recommended as **on by default**.
+
+### Specialist Class: A more specific use case that specialists could enable to get more in-depth information that they already understand how to use
+
+When instrumentation falls into the Specialist Class, we are assuming the target audience is already familiar with the concept and knows exactly what they are looking for and why. The goal for Specialist Class instrumentation is to ensure that users who have very specific and detailed needs are still covered by our semantic conventions so they don't need to go out of their way coming up with their own, risking the same kind of disparate instrumentation problem that semantic conventions are intended to solve.
+The main differences in how we handle Speciialist Class instrumentation are:
+
+1. The names and resulting values will map directly to what a user would expect hunting down the information themselves. We will rarely be prescriptive in how the information should be used or how it should be broken down. For example, a metric to represent a process's cgroup would have the resulting value match exactly to what the result would be if the user called `cat /proc/PID/cgroup`.  
+2. If a piece of instrumentation is specific to a particular operating system, the name of the operating system will be in the instrumentation name. See [Operating System in names](#operating-system-in-names) for more information. For example, a metric for a process's cgroup would be `process.linux.cgroup`, given that cgroups are a specific Linux kernel feature.
+
+### Examples
+
+Some General Class examples:
+
+* Memory/CPU usage and utilization metrics  
+* General disk and network metrics  
+* Universal system/process information (names, identifiers, basic specs)
+
+Some Specialist Class examples:
```

Comment:
While the whole description of the rationale here is exactly how it should be, I think we miss the part of having a set of rules/guidelines/sanity-checks that would help somebody in the future to decide into which directory a metric or attribute fall into. This might not be quite easy to define because of the nature of this problem but maybe it would worth adding a section in the bottom suggesting how this kind of situations should be handled in the future.

Reply:
I do have a case study below for `process.linux.cgroup`; perhaps I can adapt this to more general rules?

Reply:
Done in [487af83](https://github.com/open-telemetry/semantic-conventions/pull/1618/commits/487af83f99a9ab8eeeb937e6cfac28638114cd8b)